packages/website/src/components/Playground.module.css

```diff
@@ -3,89 +3,95 @@
   --playground-secondary-color: var(--ifm-color-emphasis-100);
 }
 
-.options {
+.playgroundMenu {
+  width: 100%;
   background: var(--playground-main-color);
   overflow: auto;
-  z-index: 1;
 }
 
-.sourceCode {
-  height: 100%;
-  border: 1px solid var(--playground-secondary-color);
+.tabCode {
+  flex: 1;
+  overflow: hidden;
 }
 
-.codeBlocks {
-  display: flex;
-  flex: 1 1 0;
-  flex-direction: row;
-  height: 100%;
-  width: calc(100vw - 20rem);
+.hidden {
+  display: none;
+  visibility: hidden;
```

Comment:
`display: none` doesn't render at all, why do you need `visibility: hidden`?

Reply:
we do not need it, i just moved this css from bellow

Reply:
```suggestion
```